html-aam/index.html

```diff
@@ -191,6 +191,69 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
           the [[core-aam-1.2]].
         </p>
       </section>
+      <section id="mapping_contextual">
+        <h3>Exposing HTML Features With Contextual Roles</h3>
+        <p>
+          An element with a <strong>contextual role</strong> is one whose computed role semantics are based on its ancestral <a>accessibility tree</a> relationship to another element or elements. Or,
+          an element whose computed role is dependent upon whether it has other associated accessibility information, such as an author-provided accessible name.
+        </p>
+        <p>
+          For instance, an element that is an <strong>accessibility child</strong> of its required <strong>accessibility parent</strong> would expose a computed role for its given context, such as a
+          list item (`li` element) within an unordered list (`ul` element). However, the same element might be used within a different markup pattern and could therefore expose different
+          contextually-dependent computed roles, such as a `section` element with an author-provided accessible name (role `region`) versus an unnamed `section` element (role `generic`).
+        </p>
+        <p>When the conditions for an element's contextual role are not met, it will commonly expose a computed role of `generic` or with a <strong>minimum role</strong>, if applicable.</p>
```

Comment:
```suggestion
        <p>When the conditions for an element's contextual role are not met, it will commonly be exposed with a computed role of `generic` or with a <strong>minimum role</strong>, if applicable.</p>
```

Grammar ^